pkg/conversion/converter.go

```diff
@@ -495,15 +495,6 @@ func (c *Converter) callCustom(sv, dv, custom reflect.Value, scope *scope) error
 // one is registered.
 func (c *Converter) convert(sv, dv reflect.Value, scope *scope) error {
 	dt, st := dv.Type(), sv.Type()
-	// Apply default values.
-	if fv, ok := c.defaultingFuncs[st]; ok {
```

Comment:
That's a fair question, and I don't know the answer, so I suspect it's no,
or at least "not directly".

On Mon, Oct 26, 2015 at 8:06 PM, Daniel Smith <notifications@github.com>
wrote:

> In pkg/conversion/converter.go
> <https://github.com/kubernetes/kubernetes/pull/16080#discussion_r43080113>
> :
>
> > @@ -495,15 +495,6 @@ func (c *Converter) callCustom(sv, dv, custom reflect.Value, scope *scope) error
> >  // one is registered.
> >  func (c *Converter) convert(sv, dv reflect.Value, scope *scope) error {
> >  	dt, st := dv.Type(), sv.Type()
> > -	// Apply default values.
> > -	if fv, ok := c.defaultingFuncs[st]; ok {
>
> @thockin <https://github.com/thockin> @wojtek-t
> <https://github.com/wojtek-t> Is there a test that would tell us if
> defaulting stopped working?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/16080/files#r43080113>.
>